.github/workflows/vale.yml

```diff
@@ -0,0 +1,34 @@
+name: Validate content with Vale
+
+on:
+  pull_request:
+    paths:
+      - 'content/**'
+
+  # Allows manual workflow run (must in default branch to work)
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the docs repo
+        uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          files: content/**
+
+      - name: Install Vale
+        env:
+          VALE_VERSION: "2.22.0"
+        run: wget https://github.com/errata-ai/vale/releases/download/v${{env.VALE_VERSION}}/vale_${{env.VALE_VERSION}}_Linux_64-bit.tar.gz && tar -xvzf vale_${{env.VALE_VERSION}}_Linux_64-bit.tar.gz
+
+      - name: Run Vale
+        if: steps.changed-files.outputs.any_changed == 'true'
+        # Run vale and display the output. Fail if the output is greater than 1 line.
+        # Vale uses a 0 return code for warnings and suggestions. These should be considered failures.
```

Comment:
Non-blocking: Are we always going to treat suggestions as failures? That feels a bit harsh for when we move to make this a mandatory step that has to pass for a PR.

Reply:
The problem is that we are inheriting the warning/suggestion/error flags from the upstream style libraries. I think it would be significant effort to apply our editorial opinions to them.

What I'd rather do is let any vale warn/suggest/error be a decision point to either:
* update the rules/style (e.g., "special")
* improve the writing (e.g., using latin) 
* Set a vale exception for that line/paragraph to ignore the rule.

This also keeps the results clean. I worry about a future where vale runs are 1400 suggestions creating noise against the real errors and problems. 

If it becomes a thing™ in the future I'd be happy to reevaluate error levels.

Reply:
This sounds like a sound approach to me.